docs/layers/accounts/deploy-accounts.mdx

```diff
@@ -75,10 +71,10 @@ This step-by-step process outlines how to deploy AWS accounts using `atmos` work
       With the addition of support for dynamic Terraform roles, our `baseline` cold start refarch layer now depends
       on/requires that we have `aws-teams` and `aws-team-roles` stacks configured. This is because `account-map` uses those
       stacks to determine which IAM role to assume when performing Terraform in the account, and almost every other component
-      uses `account-map` (indirectly) to chose the role to assume.
+      uses `account-map` (indirectly) to chose the role to assume. However, these components do _not_ need to be deployed yet.
     </Note>
 
-    Again verify the "account" configuration in `stacks/catalog/account.yaml`.
+    Again verify the "account" configuration in `stacks/catalog/account.yaml`. **This is the point of no return!**
 
```

Comment:
I don't think it's clear to a newbie what we mean, when we say the point of no return in the context of account configuration. Sounds negative. 

```suggestion
    Again verify the "account" configuration in `stacks/catalog/account.yaml`. **This is the point of no return!**

```

Reply:
rephrased. Less urgent / negative